site/content/blog/2025-01-09-zap-2-16-0/index.md

```diff
@@ -0,0 +1,111 @@
+type: post
+tags:
+- blog
+- release
+date: "2025-01-09"
+authors:
+    - simon
+---
+ZAP 2.16.0 has just been [released](/docs/desktop/releases/2.16.0/)!
+
+![ZAPbot with 2.16.0 banner](images/blog-2-16-0.png)
+
+This release was made possible thanks to [Checkmarx](https://checkmarx.com/) 
+who employ 3 of the [Core Team](/docs/team/) to work fulltime on ZAP.
+
+This is a big release and includes many very singificant changes.
+Look out for new [Blog Posts](/blog/) and [Videos](/videos/) which will cover some of these new features in much more depth in the coming days and weeks.
+
+### Update to a Minimum of Java 17
+
+ZAP now requires a minimum of Java 17 to run. This allows us to use more modern Java features in the ZAP codebase.
+
+As a result of this move scripts which use the Nashorn JavaScript engine may no longer work, 
+this is because the engine is no longer present in Java 17.
+
+Any scripts configured to use Nashorn will automatically be changed to use the 
+[Graal.js](/docs/desktop/addons/graalvm-javascript/) JavaScript engine.
+However you may still need to migrate these scripts, see the 
+[Migration Guide from Nashorn to GraalJS](https://www.graalvm.org/latest/reference-manual/js/NashornMigrationGuide/).
+
+### Client Spider
+
+The [Client Integration](/docs/desktop/addons/client-side-integration/) add-on is now included as standard, and provides a new 
+[Client Spider](/docs/desktop/addons/client-side-integration/spider/).
+The Client Spider works in a similar way to the AJAX Spider but it has access to the DOM via the ZAP Browser Extension
+which means that it can find content which the AJAX Spider cannot find.
+
+While it is still at an early stage we believe it is a more effective approach than the AJAX Spider.
+We will be focussing on improving the Client Spider and the current plan is for it to supersede the AJAX Spider
+as the recommended way of crawling modern web apps.
+
+### Detachable Tabs
+
+You can now detach any of the ZAP desktop tabs so that they will appear in their own windows.
+This means that you can see more than one tab in the group at the same time, and allows you to distribute those windows
+across multiple monitors. 
+
+Just right click any tab to see the 'detach' option and close the tab window to return it to the main ZAP window.
+
+### Standard Policy Definitions
+
+We now have a standard set of active scan policy definitions which we recommend for use in specific situations.
+Using these definitions will just enable the most suitable active scan rules. This will speed up your active scans
+and will reduce noise. Some of the standard active scan rules are really targetted at pentesters, and these can waste 
+developers time.
+
+For more details see the new [Scan Policies](/docs/desktop/addons/scan-policies/) add-on.
+
+### Site Tree Export and Pruning
+
+We now have a standard format for representing the Sites Tree in a text file: 
```

Comment:
YAML vs text?

Reply:
YAML is text 😁 Thats explained in the relevant docs, and I dont really want to change the help etc again just for consistency..